gnoblog-cli/cmd/util.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/adrg/frontmatter"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+func parsePost(reader io.Reader) (*post, error) {
+	var p post
+	rest, err := frontmatter.MustParse(reader, &p)
+	if err != nil {
+		return nil, fmt.Errorf("invalid post frontmatter: %w", err)
+	}
+
+	p.Title, err = extractTitle(p.Body)
```

Comment:
err not checked.

Reply:
Moved it so it's more clear: [9294311](https://github.com/gnolang/blog/pull/63/commits/92943113d7454e64d656346d8ead122b095d6668)